.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "OpenCodelists",
+    "features": {
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {}
+    },
+    "postCreateCommand": "/bin/bash /workspaces/opencodelists/dokku.sh"
```

Comment:
thought: We could provide `${containerWorkspaceFolder}` as an argument to `dokku.sh` here, which then avoids hardcoding the directory in `dokku.sh`.

Reply:
thought: We could provide a `postAttachCommand` script which should restart the installed server correctly when a codespace is started after stopping.